cli/cache/cache_db.rs

```diff
@@ -95,7 +96,7 @@ impl Drop for CacheDB {
       // Hand off SQLite connection to another thread to do the surprisingly expensive cleanup
       let inner = inner.into_inner().into_inner();
       if let Some(conn) = inner {
-        tokio::task::spawn_blocking(move || {
```

Comment:
Maybe we should add some clippy rules banning the use of tokio::task::spawn and spawn_blocking? (I think by using `disallowed_methods`, but not sure)

Reply:
https://github.com/denoland/deno/pull/19056#pullrequestreview-1419095438